tests/MimeTest.php

```diff
@@ -49,6 +49,38 @@ public function testWebp()
         $this->assertFileHasMimeType('sample.webp', 'image/webp');
     }
 
+    public function testDetermineDimensions()
+    {
+        $file = new \SplFileObject(__DIR__ . '/files/sample.png', 'r');
+
+        $this->assertSame([16, 16], $this->detector->determineDimensions($file));
+    }
+
+    public function testDetermineDimensionsWithRectangleImage()
+    {
+        $file = new \SplFileObject(__DIR__ . '/files/sample2.jpg', 'r');
+
+        $this->assertSame([3, 7], $this->detector->determineDimensions($file));
+    }
+
+    public function testDetermineDimensionsWithInvalidFileType()
+    {
+        $file = new \SplFileObject(__DIR__ . '/files/sample.xml', 'r');
+
+        $this->expectException(\InvalidArgumentException::class);
+
+        $this->detector->determineDimensions($file);
+    }
+
+    public function testDetermineDimensionsWithNonZeroPosition()
+    {
+        $file = new \SplFileObject(__DIR__ . '/files/sample.jpg', 'r');
+        $file->ftell();
```

Comment:
Not really a problem, just wanted to point out that this line doesn’t do anything.

Reply:
Hi @Zegnat, thank you for your reply.
You're right and I've removed this line :+1:.